src/Bitcoin/Address/Bech32.hs

```diff
@@ -1,5 +1,8 @@
+{-# LANGUAGE DuplicateRecordFields #-}
 {-# LANGUAGE MultiWayIf #-}
+{-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
I would prefer to keep `RecordWildCards` out of our codebase.  It makes the code more difficult to read and can lead to subtle bugs.  `NamedFieldPuns` eases the naming burden for record construction and pattern binding without the costs.

Reply:
Agreed, and removed. Was just being lazy and trying to type less!